docs/reference/sql/alter.md

```diff
@@ -1,13 +1,54 @@
 # ALTER
 
-`ALTER` can be used to modify any table settings or data within the table:
+`ALTER` can be used to modify any database options, table options or metadata of the table, including:
 
+* Modify database options
 * Add/Drop/Modify a column
 * Rename a table
+* Modify table options
 
-## Syntax
+## ALTER DATABASE
+
```

Comment:
```suggestion

`ALTER DATABASE` statements can be used to modify the options of databases.

```

Reply:
Not sure if it's the place. If there're more `ALTER DATABASE` stmt like rename database then we need to change it again.